docs/guide/index.md

```diff
@@ -116,7 +116,7 @@ Running `vite` starts the dev server using the current working directory as root
 In a project where Vite is installed, you can use the `vite` binary in your npm scripts, or run it directly with `npx vite`. Here are the default npm scripts in a scaffolded Vite project:
 
 <!-- prettier-ignore -->
-```json5
+```json
```

Comment:
Shiki (the new syntax highlight plugin) doesn't support `json5`, so we have to change this. `json` still does make comments correctly highlighted in page, but we could also use `js`?

If you prefer to change this, let me know 👍

Reply:
`json` sounds good to me 👍🏼